### PR TITLE
[Tour of Beam] Learning content for "Core Transforms" module

### PR DESCRIPTION
**Pull request by sirenbyte**
#22500

**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Review by kerrydc — CHANGES_REQUESTED** [review]
The markdown is still making strange line breaks. Here the circles show correct line breaks, the arrows show bad line breaks.
Can we fix this for the whole TOB?
image

File: learning/tour-of-beam/learning-content/core-transforms/additional-outputs/description.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+While `ParDo` always outputs the main output of `PCollection` (as a return value from apply), you can also force your `ParDo` to output any number of additional `PCollection` outputs. If you decide to have multiple outputs, your `ParDo` will return all the `PCollection` output (including the main output) combined. This will be useful when you are working with big data or a database that needs to be divided into different collections. You get a combined `PCollectionTuple`, you can use `TupleTag` to get a `PCollection`.
+
+A `PCollectionTuple` is an immutable tuple of heterogeneously typed `PCollection`, "with keys" `TupleTags`. A `PCollectionTuple` can be used as input or output for `PTransform` receiving or creating multiple `PCollection` inputs or outputs, which can be of different types, for example, `ParDo` with multiple outputs.
+
+A `TupleTag` is a typed tag used as the key of a heterogeneously typed tuple, for example `PCollectionTuple`. Its general type parameter allows you to track the static type of things stored in tuples.
+
+### Tags for multiple outputs
+
+```
+  .of(new DoFn<String, String>() {
```
[the remark this answers is not in the document]
Comment (olehborysevych) [reply learning/tour-of-beam/learning-content/core-transforms/additional-outputs/description.md:114]: Hey @nausharipov could you please check if this markdown issue is caused by some flutter issue or we can fix this?

[the remark this answers is not in the document]
Comment (sirenbyte) [reply learning/tour-of-beam/learning-content/core-transforms/additional-outputs/description.md]: done

File: learning/tour-of-beam/learning-content/core-transforms/additional-outputs/description.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+```
+// beam.ParDo3 returns PCollections in the same order as
+// the emit function parameters in processWords.
+below, above, marked := beam.ParDo3(s, processWords, words)
+
+// processWordsMixed uses both a standard return and an emitter function.
+// The standard return produces the first PCollection from beam.ParDo2,
+// and the emitter produces the second PCollection.
+length, mixedMarked := beam.ParDo2(s, processWordsMixed, words)
+```
```
[the remark this answers is not in the document]
Comment (sirenbyte) [reply learning/tour-of-beam/learning-content/core-transforms/additional-outputs/description.md:31]: done

File: learning/tour-of-beam/learning-content/core-transforms/additional-outputs/description.md [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+
+### Tags for multiple outputs
+
+The Go SDK doesn’t use output tags, and instead uses positional ordering for multiple output `PCollection`s.
+
+```
+// beam.ParDo3 returns PCollections in the same order as
+// the emit function parameters in processWords.
+below, above, marked := beam.ParDo3(s, processWords, words)
+
```
[the remark this answers is not in the document]
Comment (sirenbyte) [reply learning/tour-of-beam/learning-content/core-transforms/additional-outputs/description.md:25]: done

**Pull request closed without merging**